### PR TITLE
fix spelling typo and rename UIImage extension filename

### LINKED ISSUE #1300: (Good First Issue) Fix Typo, Adjust Filename, Add Documentation
In `UIImage+Extension.swift`, there is a typo: `disclosure` is misspelled `disclouser`.
The filename does not match its siblings: `Extension` vs `Extensions`.
The extension should either be `internal` or if it must remain public, public extensions should have documentation:
- At least a one line description for each type definition.
- A Doc comment (put cursor within the function name, press `cmd-option-/`)

In addition, the `#import` should be at the top of the file.

Extra credit: Combine the `UIColor+Extensions.swift` extensions into one `extension` and add brief internal doc comments to each color with hex codes so that they are easier to use within the framework.

### PR DESCRIPTION
**Pull request by isupremedyou**
What does this implement/fix?
Fixes #1300

### CODE REVIEW

File: Sources/Views/Cells/ContactMessageCell.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ open class ContactMessageCell: MessageContentCell {
     
     public enum ConstraintsID: String {
         case initialsContainerLeftConstraint
-        case disclouserRigtConstraint
+        case disclosureRigtConstraint
```
Comment (Kaspik) [inline Sources/Views/Cells/ContactMessageCell.swift]: `Rigt` vs `Right`

**Review by Kaspik — CHANGES_REQUESTED** [review]
One more typo :)

File: Sources/Views/Cells/ContactMessageCell.swift (hunk 1 above)
Comment (isupremedyou) [reply Sources/Views/Cells/ContactMessageCell.swift]: Thanks for helping me. This is my first stab at contributing to any open source project. I'm on a solo mobile dev team so somethings are still new to me!

That typo is also resolved now.

**Review by Kaspik — COMMENTED** [review]
Thanks, nice!

Nit pick: If you wanna fully complete https://github.com/MessageKit/MessageKit/issues/1300, please try to investigate the public colors/public funcs and add short docu? :)

**Comment by isupremedyou** [comment]
@Kaspik I am thinking of attempting #1301 and perhaps i can do that at the same time?

**Comment by Kaspik** [comment]
Allright, good idea. :)

**Review by Kaspik — APPROVED** [review]

**Comment by SD10** [comment]
Thank you for contributing to MessageKit! I've invited you to join the MessageKit GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/MessageKit/MessageKit/blob/master/CONTRIBUTING.md) and join the [MessageKit Slack channel](https://join.slack.com/t/messagekit/shared_invite/enQtMjI4NzIzNzMyMzU0LTEyNmYzNzBlNjNiYjAwYjY1ZjhkMmVjNDE4OGNlMDY1OTRmNzVmMzI4MTg3OWQzZTM2NGU4NTU3ZDE5NzkwNzQ). Feel free to reach out if you have any questions! :smiley:

**Pull request merged**